docs/product/dev-toolbar/setup.mdx

```diff
@@ -36,6 +36,10 @@ You need to complete two steps to get the toolbar rendered on the page:
     </html>
     ```
 
+3. Edit Project Settings in Sentry so the Toolbar is allowed to connect from your dev, staging, and production domains.
```

Comment:
is it possible to link to the settings page from here?

Reply:
I can get to `https://sentry.sentry.io/settings/projects/` but without knowing the project name i can't get closer

Reply:
ahh its project settings, not org settings. gotcha. could make it org settings?

Reply:
it's a per-project config

Reply:
orgs can have like, their main website, but also all the backend tools with very different domains. kinda a security thing

if different projects are on the same domain that's ok. the request is for a project+domain pair. So everything could have `localhost` added to each of their lists no problem.

Reply:
the security position is that: 
we're doing a login using sentry.io, so cookies and things never leave our own control. but org data is leaving sentry.io so I wanted to validate that we're exfiltrating those api responses to a domain known to the customer. This way org data only gets sent to domains the customer controls... then the sdk can use the data to render the toolbar.